ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/113646_disable-x-xss-protection/113746_disable_x-xss-protection.cy.js

```diff
@@ -0,0 +1,20 @@
+/// <reference types ='Cypress'/>
+
+Cypress._.each(['ipad-mini'], (viewport) => {
+	describe(`113746 X-xss-Protection Header on conversion ${viewport}`, () => {
+		beforeEach(() => {
+			cy.login()
+			cy.viewport(viewport)
+            
+		});
+		
+		it('TC01: should be disabled', () => {
+            cy.url().then(urlString => {
```

Comment:
The indentation has gone a bit wild here.

Reply:
This is bit strange, on local in VS Code all seems correctly indented. I have noticed these few times now not sure if anyone else faced the same. I tweak some changes and push again if it fixes the same.